packages/orbit-components/src/Modal/ModalSection/index.tsx

```diff
@@ -29,6 +29,7 @@ export const ModalSectionWrapper = ({
         "orbit-modal-section",
         "py-lg px-md lm:p-xl box-border w-full",
         "border-b-elevation-flat-border-color border-b border-solid",
+        "lm:last-of-type:rounded-b-modal lm:first-of-type:rounded-t-modal",
```

Comment:
The first section should have top rounded corners. The last section should have bottom rounded corners.